nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -174,6 +174,8 @@ impl ServiceCore for DisseminateService {
                     .is_err()
                 {
                     tracing::error!("Timeout reached, check the logs for additional details");
+                    service_state.overwatch_handle.shutdown().await;
+                    std::process::exit(1);
                 }
```

Comment:
It seems that we need to exit the process even if the `disseminate()` returns an error (within the timeout period). If not, the test will be passed.